sgx/epid-types/src/platform_info.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+//! Platform Info Blob wrapper
+
+use mc_sgx_core_types::impl_ffi_wrapper;
+use mc_sgx_epid_types_sys::sgx_platform_info_t;
+
+/// The size of a [PlatformInfo]'s x64 representation, in bytes.
+pub use mc_sgx_epid_types_sys::SGX_PLATFORM_INFO_SIZE as PLATFORM_INFO_SIZE;
+
+/// A structure containing a "platform info blob", used by IAS
+#[derive(Default)]
+#[repr(transparent)]
+pub struct PlatformInfo(sgx_platform_info_t);
+
+impl_ffi_wrapper! {
+    PlatformInfo, sgx_platform_info_t, PLATFORM_INFO_SIZE, platform_info;
+}
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use bincode::{deserialize, serialize};
+
+    #[test]
+    fn serde() {
```

Comment:
Are there currently tests for extracting various info from PIBs? I don't know if we've cached enough PIB responses to fill out those unittests, but that could be useful for if we wanted to implement richer logic around possible PIB responses - I see the UpdateInfo tests that extract and display the data, so that's probably sufficient.

Reply:
Intel has intentionally left it as an opaque blob, so I don't think we can do that. Even UpdateInfo is only crated by calling the SDK against a PIB.

Reply:
👍